composer.json

```diff
@@ -95,7 +95,9 @@
         "twilio/sdk": "^4.10",
 
         "stack/run": "^1.0",
-        "stack/builder": "^1.0"
+        "stack/builder": "^1.0",
+
+        "piwik/device-detector": "3.7.0"
```

Comment:
Sorry for the composer.lock confusion. Let's run `composer require piwik/device-detector:~3.7.0` so this dependency gets added to composer.lock but without updating all the other dependencies. Then commit composer.lock.